### DISCUSSION TITLE
my student education pack never never get passed

### DISCUSSION THREAD

**Discussion by xiaonaixi** [community/community#54839] [GitHub Education]
### Hi everyone,

I have applied for accreditation several times and my documents include: student card and translated documents, letter of admission and translation, transcripts, and certification by the Department of Education. I have written to your company but have not received any reply. I hope your company can respond to me, I really need this help to study and improve myself!
I have attached my materials to the email, but you have been rejecting my application by the robot. I have applied 67 times! I don't know why you keep asking me to apply again.

**Comment by coolacpc** [comment]
Welcome to the club!
